src/App.js

```diff
@@ -41,28 +48,55 @@ class App extends Component {
       ...foundActivity,
       isBookmarked: !foundActivity.isBookmarked,
     }
-    console.log(newObject)
 
     this.setState({
       activities: [...startOfNewArray, newObject, ...endOfNewArray],
     })
   }
+  toggleFilter() {
+    this.setState({
+      filter: !this.state.filter,
+    })
+  }
 
   render() {
     return (
       <Grid>
         <Title>Discover</Title>
-        <List>
-          {this.state.activities.map(activity => {
-            return (
-              <ActivityItem
-                text={activity.activity}
-                isBookmarked={activity.isBookmarked}
-                onBookmark={() => this.bookmark(activity.id)}
-              />
-            )
-          })}
-        </List>
+        {this.state.filter === true ? (
```

Comment:
Der Übersichtlichkeit würde ich hier vllt mit einem IF - ELSE arbeiten, aber das musst du entscheiden. Da this.state.filter boolean ist, sollte es eigentlich auch ohne das === true funktionieren